src/profile/helpers/editData.ts

```diff
@@ -127,7 +159,7 @@ function create(
   };
 }
 
-function pickSources(
+export function pickSources(
```

Comment:
First usage of this function inside this file is pretty far away from the definition 162 -> 362. I wonder if it's possible to move it closer to usage to help readability? Sometimes can be hard if there's many functions that if you move one function closer, then some other function will be more far away 😅

Reply:
Referring to the Team's Slack discussion, ordering functions and keeping them in logical order for everyone is a tedious job and hard to keep the ordering in the long run.

We should write the code so functions are in clear, logical order and try to keep them so, but it is not required and enforced.